src/components/user/repo/user.repo.integration.test.js

```diff
@@ -88,4 +91,27 @@ describe('user repository', () => {
 			});
 		});
 	});
+
+	describe('createTombstoneUser', () => {
+		it('should return a user object', async () => {
+			const school = await testObjects.createTestSchool();
+
+			const tombstoneUser = await userRepo.createTombstoneUser(school._id, TOMBSTONE_SCHOOL_ID);
+
+			expect(tombstoneUser).to.be.an('object');
+			expect(tombstoneUser).to.contain.keys('email', 'firstName', 'lastName');
```

Comment:
Please check all tombstoneUser properties... including schoolId, birthday etc.
Please check if the tombstone school is updated properly

Reply:
done.